### ISSUE TITLE
onnx support

### ISSUE THREAD

**Issue by xhzhao** [SeanNaren/deepspeech.pytorch#344]
We know that onnx is very popular AI-model standard, and many pretrained modes are released in this [model zoo](https://github.com/onnx/models). But, i could not find any RNN model in the model list. 

Is is possible for us to push the DS2 pretrained model to the onnx model zoo?
Is there any obstacle for us to do it? say CTC loss?

**Comment by ryanleary** [comment]
CTC isn't an obstacle, and it is _possible_ but it is a challenge. I had a branch that can do it, but there were a number of hacks involved. I'll have to see if I can dig it up.

**Comment by miguelvr** [comment]
@ryanleary Native PyTorch CTC Loss is out in PyTorch 1.0rc1 https://pytorch.org/docs/master/nn.html#torch.nn.CTCLoss

Maybe it will make things easier?

**Comment by ryanleary** [comment]
No. The loss function doesn't matter for model export purposes.

**Comment by PCerles** [comment]
@ryanleary any chance you dug up that branch? Hoping to do some exporting to caffe2.

**Comment by ryanleary** [comment]
@mark-arm thanks for adding that patch. It looks like a number of the issues that existed in older PyTorch has been fixed with the exception being the RNN bias bug (https://github.com/pytorch/pytorch/issues/7871). We can probably monkey-patch the model with zero-bias tensors in this script to make export succeed.

**Comment by PCerles** [comment]
@mark-arm thank you! The RNN bias bug was not evident in the errors I was getting when I tried to export. Do you have the diff to the ONNX export code? Otherwise @ryanleary 's monkey-patch suggestion seems pretty straightforward.

**Comment by mark-arm** [comment]
@PCerles @xinli94 That's the error I fixed in the PyTorch ONNX export code. I attached my patch here: https://github.com/pytorch/pytorch/issues/7871#issuecomment-442464763

**Comment by xinli94** [comment]
@mark-arm Cool thanks! It works for me!
